src/main/java/bio/terra/cli/serialization/userfacing/input/CreateBqDatasetParams.java

```diff
@@ -13,7 +14,7 @@
 public class CreateBqDatasetParams {
   public final CreateResourceParams resourceFields;
   public final String projectId;
-  public final String datasetId;
+  public final @Nullable String datasetId;
```

Comment:
nit: I don't think you need to include `@Nullable` in these serialization classes, because all fields are nullable. ditto below. alternatively, we could always include these annotations in the serialization classes as well, but I think it should be consistent. it seems easier to just maintain these requirements on the `businessobject` package classes instead of also on their serialization POJO counterparts.